scripts/train_agents.py

```diff
@@ -6,399 +6,14 @@
 from oai_agents.common.learner import LearnerType
 from oai_agents.common.curriculum import Curriculum
 
-from utils import (get_SP_agent,
+
+from scripts.scripts.utils import (get_SP_agent,
                     get_FCP_agent_w_pop,
                     get_eval_types_to_load,
                     get_N_X_FCP_agents,
                     get_N_X_SP_agents,
                     get_agent_play_w_adversarys
                     )
-def InitializeAdversaryPlay(
```

Comment:
Can you also delete the unused files from train_helper.py and teammates_collection.py